rj_gameplay/stp/rc.py

```diff
@@ -0,0 +1,64 @@
+"""This module contains data structures that are robocup specific, ie. Robot, Ball,
+WorldState"""
+
+from typing import List
+
+import numpy as np
+
+RobotId = int
+
+
+class Robot:
```

Comment:
We should make it explicit that this is the state of the robot and not a place to apply actions to the robot.

Reply:
Done.